client/webpack.config.js

```diff
@@ -3,12 +3,15 @@ const path = require('path');
 const webpack = require('webpack');
 const autoprefixer = require('autoprefixer');
 const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const failPlugin = require('webpack-fail-plugin');
 
 
 // Always enabled plugins
 let plugins = [
   // Extract CSS files to the 'bundle.css'.
-  new ExtractTextPlugin('_bundle.css')
+  new ExtractTextPlugin('_bundle.css'),
+  // This plugin should be always required. See https://github.com/webpack/webpack/issues/708
+  failPlugin,
```

Comment:
I'm asking this because I'm not sure : Does this plugin work regardless of the position of itself within plugins array?